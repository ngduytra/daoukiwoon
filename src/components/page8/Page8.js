import React from 'react';
import './page8.styles.scss'
import Card from '../card/Card'

const dummyData = [
    {
        title: 'Free Test',
        subtitle: 'Organize across all apps by hand',
        price: '0',
        currency: '$'
    },
    {
        title: 'Low Price',
        subtitle: 'Monthly Fixed Amount',
        price: '200.000',
        currency: '$'
    },
    {
        title: 'Easy Using Methods',
        subtitle: 'Various Manuals',
        price: '200.000',
        currency: '$'
    },
    {
        title: 'Verified IT Service',
        subtitle: 'On sale in 4 countries',
        price: '300.000',
        currency: 'VND'
    }
]

const Page8 = () => {
    return (
        <div id='index-8'>
            {
                dummyData.map((data, index)=>(
                    <Card data={data} key={index}/>
                ))
            }
        </div>
    )
}

export default Page8;