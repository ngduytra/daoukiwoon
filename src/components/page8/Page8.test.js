import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from 'react';
import Page8 from './Page8';

configure({adapter: new Adapter()});

it('expect to render Page8 component', ()=>{
    expect(shallow(<Page8/>)).toMatchSnapshot();
})