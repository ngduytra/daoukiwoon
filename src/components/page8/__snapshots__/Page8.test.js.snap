// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Page8 component 1`] = `ShallowWrapper {}`;
